Apps/WebClient/src/ClientApp/test/stubs/store/options.ts

```diff
@@ -0,0 +1,48 @@
+import "reflect-metadata";
+
+import { voidMethod } from "@test/stubs/util";
+import { injectable } from "inversify";
+
+import { GatewayStoreOptions } from "@/store/types";
+
+import authStub from "./authStub";
+import commentStub from "./commentStub";
+import configStub from "./configStub";
+import encounterStub from "./encounterStub";
+import errorBannerStub from "./errorBannerStub";
+import idleStub from "./idleStub";
+import immunizationStub from "./immunizationStub";
+import laboratoryStub from "./laboratoryStub";
+import medicationStub from "./medicationStub";
+import navbarStub from "./navbarStub";
+import noteStub from "./noteStub";
+import timelineStub from "./timelineStub";
+import userStub from "./userStub";
+
+@injectable()
+export default class StoreOptionsStub implements GatewayStoreOptions {
```

Comment:
We have both `options.ts` and `storeOptionsStub.ts` and need to remove one.  I'd prefer keeping `storeOptionsStub.ts`.

Reply:
agreed, is part of the automated merge

Reply:
I see both files still in the latest commit on this branch:

https://github.com/bcgov/healthgateway/tree/features/10460_b/Apps/WebClient/src/ClientApp/test/stubs/store

Reply:
Manually removed it now.